oem/ibm/libpldmresponder/inband_code_update.cpp

```diff
@@ -314,13 +321,40 @@ void CodeUpdate::setVersions()
             if (interface.first == "xyz.openbmc_project.Software.Activation")
             {
                 auto imageInterface = "xyz.openbmc_project.Software.Activation";
+
                 auto imageObjPath = path.str.c_str();
 
-                // If the version interface is added with the Activation
-                // value as Invalid, it is considered as assemble code
-                // update image failure.
-                // Failure is occured while assembling
-                // lid files or while creating a tar ball.
+                // If the
+                // version
+                // interface
+                // is
+                // added
+                // with
+                // the
+                // Activation
+                // value
+                // as
+                // Invalid,
+                // it is
+                // considered
+                // as
+                // assemble
+                // code
+                // update
+                // image
+                // failure.
+                // Failure
+                // is
+                // occured
+                // while
+                // assembling
+                // lid
+                // files
+                // or
+                // while
+                // creating
+                // a tar
+                // ball.
```

Comment:
what happened to this block?

Reply:
this is caused by clang, i retried running clang but there is no change to this

Reply:
Done, it was due to clang.